### PR TITLE
BarChar and bubbleTree has been added

### PR DESCRIPTION
**Pull request by borysyuk**

### CODE REVIEW

File: src/bindings/angular/bubbletree/index.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import BubbleTreeComponent from '../../../components/pie'
```
Comment (pwalsh) [inline src/bindings/angular/bubbletree/index.js]: Import from pie?

File: src/bindings/angular/chart/index.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import ChartComponent from '../../../components/pie'
```
Comment (pwalsh) [inline src/bindings/angular/chart/index.js]: Import from pie?

File: src/components/pie/index.js [hunk 3]
```diff
@@ -14,11 +14,11 @@ class PieChartComponent extends events.EventEmitter {
   }
 
   refresh() {
-    var bounds = wrapper.getBoundingClientRect();
-    this.chart.resize({
-      height: bounds.height,
-      width: bounds.width
-    });
+    var bounds = this.wrapper.getBoundingClientRect();
+    //this.chart.resize({
```
Comment (pwalsh) [inline src/components/pie/index.js]: ?

**Comment by pwalsh** [comment]
@borysyuk see above. some bad imports? code itself looks good.

**Comment by borysyuk** [comment]
@pwalsh fixed

**Comment by pwalsh** [comment]
@borysyuk ok, good to merge

**Pull request merged**